igvm/hypervisor_preferences.py

```diff
@@ -130,6 +130,36 @@ def __call__(self, vm, hv):
         return value is not None and value > self.limit
 
 
+class HypervisorCpuUsageLimit(object):
+    """Check for CPU usage of the hypervisor incl. the predicted CPU usage
+    of the VM to be migrated.
+
+    Dismiss any hypervisor, which would be above its threshold.
+    """
+    def __init__(self, hardware_model, hv_cpu_thresholds):
+        self.hardware_model = hardware_model
+        self.hv_cpu_thresholds = hv_cpu_thresholds
+
+    def __repr__(self):
+        args = repr(self.hardware_model) + ', ' + repr(self.hv_cpu_thresholds)
+
+        return '{}({})'.format(type(self).__name__, args)
```

Comment:
Maybe stick to either format or string concatenation ?

Reply:
I would like to leave it like it is, since all our Preferences are looking the same.